memory.js

```diff
@@ -0,0 +1,70 @@
+var content = ['blk10','blk10','blkace','blkace','blkjack','blkjack','blkjoker','blkjoker','blkking','blkking','blkqueen','blkqueen','red10','red10','redace','redace','redjack','redjack','redjoker','redjoker','redking','redking','redqueen','redqueen'];
+var values = [];
+var card_ids = [];
+var cardsFlipped = 0;
```

Comment:
Minor: be consistent about whether you're using camelCase (more common in JS, for normal variables) or under_scores.